src/profile/helpers/updateMutationVariables.ts

```diff
@@ -1,4 +1,4 @@
-import { isEqual } from 'lodash';
+import _, { isEqual } from 'lodash';
```

Comment:
Should we import the pick separately like in the case of isEqual? Or just '_' to handle both?

Reply:
Auto-import did this. _.isEqual is better.

Reply:
Done.